tools/wpt/browser.py

```diff
@@ -643,10 +680,11 @@ def find_binary(self, venv_path=None, channel=None):
         raise NotImplementedError
 
     def find_webdriver(self, channel=None):
-        raise NotImplementedError
+        return find_executable("chromedriver")
```

Comment:
`ChromeiOS` does not use `chromedriver` and is also in the prototype phase, so this (and `install_webdriver`) should `raise NotImplementedError`.

Reply:
ChromeiOS is not going to inherit from ChromeAndroidBase, while AndroidWebView is going to and use this function.

Reply:
No, it does not; it inherits from `Browser`, which isn't changed here, and we are not using or inheriting from `ChromeiOS` in `ChromeAndroidBase` or `AndroidWebView`, so I'm not sure why this needs to be changed.

Reply:
Oops, I didn't intend to change this part. ChromeiOS change has been removed now.